### PR TITLE
Add effects for Xin Yue's Field of Past Regrets and Near-Death Experience

### PR DESCRIPTION
**Pull request by simonward**

### CODE REVIEW

File: packs/bestiary-effects/effect-field-of-past-regrets.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
+    "_id": "y6fibVDs9QF2vbwu",
+    "img": "icons/magic/unholy/orb-swirling-pink.webp",
+    "name": "Effect: Field of Past Regrets",
+    "system": {
+        "description": {
+            "value": "<p>Creatures who have at least one past life gain a +1 status bonus to saving throws against mental effects. Living, non-nindoru creatures gain weakness 3 to mental damage.</p>"
```
Comment (simonward) [inline packs/bestiary-effects/effect-field-of-past-regrets.json:7]: Not sure if this is a spoiler. If not, we could also add a RollOption toggle.

Comment (TikaelSol) [reply packs/bestiary-effects/effect-field-of-past-regrets.json:7]: Just have it always grant the +1

Comment (TikaelSol) [inline packs/bestiary-effects/effect-field-of-past-regrets.json:7]:
```suggestion
            "value": "<p>PCs gain a +1 status bonus to saving throws against mental effects. Living, non-nindoru creatures gain weakness 3 to mental damage.</p>"
```

File: packs/bestiary-effects/effect-field-of-past-regrets.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        "publication": {
+            "license": "OGL",
+            "remaster": false,
+            "title": "Pathfinder #197: Let the Leaves Fall"
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "has-past-life",
```
Comment (TikaelSol) [inline packs/bestiary-effects/effect-field-of-past-regrets.json:27]:
```suggestion
                    "self:type:character",
```

**Pull request merged**